docs/examples/basic_conceptual_guide/Part_3-model_analyzer/Readme.md

```diff
@@ -0,0 +1,108 @@
+
+In this process, there are three major contributors of latency:
+* Network Latency
+* Inference Compute time
+* Latency caused due to wait time in the model's queue
+
+Minimizing **Network latency** is a case by case process. For instance, let's consider computer vision models. These models use image frames, videos and 3D data like point clouds which can be quite large in size, thus requiring higher bandwidth to move. Most images are saved in float32 format, which can be converted to float16 format. This may affect the dynamic range in the image, which may or may not affect model performance based preprocessing steps taken while training the model but can definitely reduce the latency as less data needs to be transmitted.
+
+Accelerating models to trim down the effective **compute time** is commonly achieved with a plethora of techniques like: optimizing the network graphs by fusing layers, reducing models precision, fusing kernels and more! This topic is covered in much more depth in Part 4 of this series.
+
+Latency in **queues** can primarily be addressed by adding more instances of the model. This may or may not require additional resources based on the GPU utilization of the current number of instances. This is the core utilization problem that needs to be solved specifically for each deployment environment. To streamline this experience, Triton Inference Server comes with Model Analyzer.
+
+Model Analyzer is a CLI tool to help with a better understanding of the compute and memory requirements of the Triton Inference Server models by sweeping through configurations settings and generating reports summarizing performance.
+
+With Model Analyzer users can: 
+* Run customizable configuration sweeps can be done to identify the best possible configuration for the expected workload and hardware. 
+* Summarize findings about latency, throughput, GPU resource utilizations, power draw and more, with detailed reports, metrics and graphs. These reports help compare performance across different configurations of setup.
+* Tailor model deployments to cater to user's Quality of Service requirements like specific p99 latency limits, GPU memory utilization and more!
+
+## Using Model Analyzer
+ 
+### Pre-requisites
+
+Refer to Part 2 of this series to get access to the models. Refer to the Model Analyzer [installation guide](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/install.md#recommended-installation-method) for more information about installing Model Analyzer. For ease of following along, use these commands to install model analyzer:
+
+```
+sudo apt-get update && sudo apt-get install python3-pip
+sudo apt-get update && sudo apt-get install wkhtmltopdf
+pip3 install triton-model-analyzer
```

Comment:
We have been pushing people towards using the containers instead of pip install. Are you telling people to manually install triton server? Or to use the container? If the container, then you should be doing the same here for MA.

Reply:
I still link to the docker instructions. I am including pip here as it is easier to follow along with pip than spinning up a different container.

Reply:
We made a conscious decision to put our docker instructions first and to use docker mode inside because they were cleaner/easier, it made sure all versions were consistent, and it made it easier for us to triage and support issues.  It still seems like we should stay consistent with that.

I assume you are telling people to use docker for the server? Or are you having them manually install the server?